### ISSUE TITLE
React: Consider consistent naming for subgraph configuration

### ISSUE THREAD

**Issue by andy-hook** [aragon/connect#295]
I think it would be nice if the connector configuration for the provider matched those expected by the hooks. For example, I recently fat fingered the org subgraph config as it expected `orgSubgraphUrl` while the hooks expect `subgraphUrl`.

A small detail, and one easily caught in a statically typed project, but just my 2c!

**Comment by bpierre** [comment]
Agreed, not only for React but for the main library too: we should use `subgraphUrl`.
